### DISCUSSION TITLE
Making masking configurable

### DISCUSSION THREAD

**Discussion by johnny-duo** [airbnb/lottie-ios#2007] [General]
Hi, I'm trying to open a PR making the behavior from here configurable: https://github.com/airbnb/lottie-ios/pull/1814

I think the default behavior being true makes sense but the switch is breaking a bunch of animations for me which is hard to debug across a large range of devices. Could I make this configurable? If so what would be best practice for doing this? I was thinking of adding it to LottieConfiguration.

**Comment by calda** [comment]
Would it be ok for your use case if we exposed a public `maskAnimationsToBounds` property on `LottieAnimationView`? Since this is a somewhat niche legacy thing to want to configure, it would be nice if we could avoid elevating it all the way to the global `LottieConfiguration` type.

**Reply by johnny-duo** [reply]
Yup, that makes sense! Will you be implementing it? If not I can open a PR today

**Reply by calda** [reply]
Could you post a PR? That would be helpful :)
